docs/src/config/packages.md

```diff
@@ -36,7 +36,8 @@ Each package, whether it's defined in the `[package]` section or in the `[packag
 A package, by Knope's definition, has a single version. There can, however, be multiple files which contain this version (e.g., `Cargo.toml` for a Rust crate and `pyproject.toml` for a Python wrapper around it). As such, you can define an array of `versioned_files` for each package as long as they all have the same version and all are supported formats. If no file is included in `versioned_files`, the latest Git tag in the format created by the [`Release`] step will be used. The file must be named exactly the way that `knope` expects, but it can be in nested directories. The supported file types (and names) are:
 
 1. `Cargo.toml` for Rust projects
-2. `pyproject.toml` for Python projects (using [Poetry's metadata](https://python-poetry.org))
+2. `pyproject.toml` for Python projects (using [PEP 621](https://packaging.python.org/en/latest/specifications/declaring-project-metadata/) or [Poetry's metadata](https://python-poetry.org))
+   - If both are present, `project.version` takes precedence over `tool.poetry.version`.
```

Comment:
Wanted to call this bullet point out for discussion specifically.

I don't think there is ever a situation where someone would want different versions in these two properties and `knope` already throws errors if it detects mismatched versions on different versioned files so I think the options here are to prefer one over the other or detect mismatches and let the user know. 

And now that I'm writing this, if we prefer the new `project.version` with this code, it would present a problem for `knope` users who are using `tool.poetry.version` but happen to have a stale `project.version`. When they update, they might unexpectedly bump based on the stale `project.version` (or see an error if they have multiple versioned files). 

Curious what your take on all this is @dbanty